arithmetization/src/test/java/net/consensys/linea/zktracer/module/oob/OobCallTest.java

```diff
@@ -251,7 +253,8 @@ private void testRecursiveCalls(EWord iterations, int numberOfOnesInOobEvent1) {
         ToyExecutionEnvironmentV2.builder()
             .accounts(List.of(userAccount, contractCallerAccount))
             .transaction(tx)
-            .testValidator(x -> {})
+            .transactionProcessingResultValidator(
```

Comment:
Can't you use a default instead of adding all these invocations?  Or is that you want the default not to be the `EMPTY_VALIDATOR` generally, but in all these cases (for some reason) you don't?

Reply:
I just preserve the same behaviour as before. Earlier it was `.testValidator(x -> {})` which is essentially same as empty validator.